### PR TITLE
Reset scanner cache validity period to 12h

### LINKED ISSUE #2115: Vulnerability Analysis Cache expiry erroneuously reduced to 14.4 minutes
### Current Behavior:
The Vulnerability Analysis Cache expiry time was reduced from 24h to 14.4 minutes by mistake in #1841

[https://github.com/DependencyTrack/dependency-track/commit/ac6186cd3f7c8d3731cac5ba944732df3ba6fda4#diff-4a2691c1673ee2620[…]62f4d1d6eada58a83596L73](https://github.com/DependencyTrack/dependency-track/commit/ac6186cd3f7c8d3731cac5ba944732df3ba6fda4#diff-4a2691c1673ee2620dd3bec2cd3a914876aa86233a0562f4d1d6eada58a83596L73)
[https://github.com/DependencyTrack/dependency-track/commit/ac6186cd3f7c8d3731cac5ba944732df3ba6fda4#diff-c0b9dee940783572a[…]4df4e138fef3381477e0R49](https://github.com/DependencyTrack/dependency-track/commit/ac6186cd3f7c8d3731cac5ba944732df3ba6fda4#diff-c0b9dee940783572a4f96155103f4d69dffc9b155c744df4e138fef3381477e0R49)

### Steps to Reproduce:
Not needed

### Expected Behavior:
Set cache expiry back to 24h, or as discussed on Slack 12h might be better: https://owasp.slack.com/archives/C6R3R32H4/p1667471703846329

### Additional Details:
This needs a database migration to convert the wrong default cache expiry currently stored in the database.
A text update might be good for the Task Schedule page in the UI to explain that the Cache Analysis Cleanup Cadence is a different setting that completely empties the cache.
In a future PR it might be worthwhile to make the cache expiry property configurable, maybe with some kind of sane/safe minimum value to avoid overloading the analyzer APIs.

### PR DESCRIPTION
**Pull request by nscuro**
Fixes #2115

Signed-off-by: nscuro <nscuro@protonmail.com>

### CODE REVIEW

File: docs/_docs/analysis-types/known-vulnerabilities.md [hunk 1]
```diff
@@ -49,4 +49,4 @@ VulnDB is a source of vulnerability intelligence that provides its own content.
 
 Dependency-Track contains an internal limiter which prevents repeated requests to remote services when performing
 vulnerability analysis. When a components Package URL or CPE is successfully used for a given analyzer, the action
-and the timestamp is recorded and compared to the interval throttle. The interval throttle defaults to 24 hours.
+and the timestamp is recorded and compared to the interval throttle. The interval throttle defaults to 12 hours.
```
Comment (valentijnscholten) [inline docs/_docs/analysis-types/known-vulnerabilities.md:52]: Can we rewrite to say analyzer results are cached? That would be more clear than "the action is being recorded" and "interval throttle".

_Dependency-Track contains an internal limiter which prevents repeated requests to remote services when performing
vulnerability analysis. When a components Package URL or CPE is successfully analyzed by a given analyzer, the result is cached. By default these cache entries expiry after 12h._

Comment (nscuro) [reply docs/_docs/analysis-types/known-vulnerabilities.md]: Good point, I updated this part according to your suggestion. Thanks!

**Review by stevespringett — APPROVED** [review]
LGTM

**Comment by nscuro** [comment]
Will merge and release tomorrow morning CET.

**Pull request merged**